packages/lesswrong/components/shortform/ShortformSubmitForm.jsx

```diff
@@ -1,26 +1,62 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import { Components, registerComponent } from 'meteor/vulcan:core';
 import { withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
 
 const styles = theme => ({
   root: {
     marginLeft: 12,
     marginRight: 12
+  },
+  placeholderWrapper: {
+    marginTop: 12,
+    marginBottom: 24,
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  placeholder: {
+    ...theme.typography.commentStyle,
+    ...theme.typography.body2,
+    fontWeight: 300,
+    color: theme.palette.grey[600]
+  },
+  placeholderSubmit: {
+    ...theme.typography.commentStyle,
+    ...theme.typography.body2,
+    color: theme.palette.grey[500],
+    marginRight: 8
   }
 }) 
 
-const ShortformSubmitForm = ({ classes, successCallback}) => {
-  const { CommentsNewForm } = Components;
+class ShortformSubmitForm extends PureComponent {
+  state = { expanded: false }
```

Comment:
Rather than turn this into a class for this one state variable, this would be a good place for a `useState` hook.

Reply:
But also: What is this and what does it have to do with ungating single-line comments?